### PR TITLE
BaSP-TG-144: Edit/Create admin (SuperAdmin)

### PR DESCRIPTION
**Pull request by Ansitox**
- Create form to create/edit admins
- Once created/edited redirect to the super admin home

QA: 
How to test?:
How to test?:
Log in as a Super Admin (email: [oclacep@wikimedia.org](mailto:oclacep@wikimedia.org), password: prueba1234)
When logged you should se a nav bar (temporary), move to the first "form" link (Add admin form)
You should see a form with empty inputs.
The "cancel" button must redirect you to the page you were before.
The "reset" button shuod clear the fields if you wrote something on them.
Once completed the fields the "confirm" button should open a modal to confirm the creation of a new admin, On cancel, mdoal will close. On confirmyou should create the new user, and a success modal will appear.
Move to home link again.
Now move to the second "form" link (Update admin form).
You should see a form with filled inputs (this is a hardcoded id, so the fields will be always the same)
The cancel and reset buttons should work as in the add admin form.
To test the "confirm" button change some of the fields (don't change the email, and you NEED to change the password (please change the password to "prueba1234" once you've finished with the tests).
Note: The nav bar is temporaryand just to use on the test of the PRs, any bad function on navigation should be ignored.

### CODE REVIEW

**Review by matescarabino — APPROVED** [review]
LGTM

File: src/Components/SuperAdmins/NavBar/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+const NavBar = () => {
+  const dispatch = useDispatch();
+  const { modalContent, showConfirmModal } = useSelector((state) => state.auth);
+  const onConfirm = () => {
+    dispatch(logout());
+    dispatch(confirmModalClose());
+  };
+
+  const onSubmit = () => {
+    const content = `Are you sure you want to logout?`;
```
Comment (CristianBucci) [inline src/Components/SuperAdmins/NavBar/index.js:17]: Change Backticks!

**Review by CristianBucci — APPROVED** [review]
Everything looks right!

File: src/Components/SuperAdmins/NavBar/index.js (hunk 1 above)
Comment (Ansitox) [reply src/Components/SuperAdmins/NavBar/index.js]: done

**Review by julianpiaggio — APPROVED** [review]
🕵️‍♂️ QA FEEDBACK

We wanted to tell you that we consider that your pr complies
the requirements to be approved, and that it can be merged into the master branch 👍

**Pull request merged**